src/tools/orb_radar.rs

```diff
@@ -291,4 +300,27 @@ impl OrbRadar {
             OrbSource::Chest => state.settings.color_orb_chests,
         }
     }
+
+    fn collected_orbs(state: &mut AppState) -> HashSet<i32> {
+        let world = state
+            .noita
+            .as_mut()
+            .and_then(|n| n.get_world_state().unwrap_or(None));
+        let mut collected_orbs = HashSet::new();
+        if world.is_none() {
+            return collected_orbs;
+        }
+        let orbs_found_thisrun = &world.unwrap().orbs_found_thisrun;
```

Comment:
ew ew ewww

```rust
let Some(world) = world else {
    return collected_orbs;
};
```
is one of the ways (fancy relatively new syntax)

the classic way is
```rust
let world = match world {
    Some(world) => world,
    None => return collected_orbs,
};
```

Because the type of `return` (and some other things) statement is divergent, so the type of resulting `world` is a nice T from the Option<T>.

Using is_some/is_none + unwrap in languages with any sort of pattern matching is like ouch :joy: